k8s/target.yml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: symmetricds-target
+  labels:
+    name: symmetricds-target
+    role: service
+spec:
+  ports:
+  - name: http
+    port: 80
+    targetPort: 31415
+  selector:
+    name: symmetricds-target
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: symmetricds-target
+  labels:
+    name: symmetricds-target
+    role: deployment
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: symmetricds-target
+        role: pod
+    spec:
+      containers:
+        - name: symmetricds
+          image: quay.io/ukhomeofficedigital/symmetricds:v0.1.2
+          env:
+            - name: HTTPS
+              value: 'FALSE'
+            - name: GROUP_ID
+              value: 'target'
+            - name: DB_TYPE
+              value: 'postgres'
+            - name: DB_HOST
+              value: 'postgres-host-name'
+            - name: DB_NAME
+              value: 'target_database'
+            - name: DB_USER
+              valueFrom:
+                secretKeyRef:
+                  name: db-creds
+                  key: username
+            - name: DB_PASS
+              valueFrom:
+                secretKeyRef:
+                  name: db-creds
```

Comment:
We should include the secret for this explicitly. Obviously the secret wouldn't normally go in source control but this is just to demonstrate the principle.